cpp/oneapi/dal/algo/subgraph_isomorphism/backend/cpu/inner_alloc.hpp

```diff
@@ -54,16 +51,16 @@ struct inner_alloc {
         return ptr;
     }
 
-    detail::byte_alloc_iface* get_byte_allocator() {
+    byte_alloc_iface_t* get_byte_allocator() {
         return byte_allocator_;
     }
 
-    const detail::byte_alloc_iface* get_byte_allocator() const {
+    const byte_alloc_iface_t* get_byte_allocator() const {
         return byte_allocator_;
     }
```

Comment:
In general, type of returned value is not connected to the function const qualifications.  

Suggest to remove both overloads and have:
```c++
byte_alloc_iface_t* get_byte_allocator() const {
    return byte_allocator_;
}
```